hudi-client/hudi-spark-client/src/main/scala/org/apache/hudi/SparkFileFormatInternalRowReaderContext.scala

```diff
@@ -55,13 +54,13 @@ import scala.collection.mutable
  *
  * This uses Spark parquet reader to read parquet data files or parquet log blocks.
  *
- * @param parquetFileReader A reader that transforms a [[PartitionedFile]] to an iterator of
+ * @param fileReader A reader that transforms a [[PartitionedFile]] to an iterator of
  *                          [[InternalRow]]. This is required for reading the base file and
  *                          not required for reading a file group with only log files.
  * @param filters           spark filters that might be pushed down into the reader
  * @param requiredFilters   filters that are required and should always be used, even in merging situations
  */
-class SparkFileFormatInternalRowReaderContext(parquetFileReader: SparkParquetReader,
+class SparkFileFormatInternalRowReaderContext(fileReaders: java.util.Map[HoodieFileFormat, SparkFileReader],
```

Comment:
@yihua : is this a requirement that we need to solve? 
i.e. even within one spark stage, some spark task might do format1, while others might do format2?